backend/battle/tests/test_view.py

```diff
@@ -1,15 +1,104 @@
-from django.test import TestCase
 from model_bakery import baker
-from battle.models import PokemonTeam
+from common.utils.tests import TestCaseUtils
+from django.urls import reverse
+from battle.models import Battle, PokemonTeam
 from battle.battles.battle import get_winner_for
 
 
-class GetWinnerTest(TestCase):
+class ListBattlesTest(TestCaseUtils):
     def setUp(self):
         super().setUp()
-        self.creator, self.opponent = baker.make('users.User', _quantity=2)
-        battle = baker.make('battle.Battle', creator=self.creator, opponent=self.opponent)
-        self.team_creator = baker.make('battle.Team', battle=battle, trainer=self.creator)
+        self.battle = baker.make('battle.Battle', creator=self.user)
+        self.battle_from_user = Battle.objects.filter(creator=self.user)
```

Comment:
I'm seeing that you are using this to avoid making queries on tests. I would like to discourage this, especially because it's not clear that each test needs this specific query. Remember "Explicit is better than implicit". Another issue with this will be repeat for all tests even the ones that don't require this query

Reply:
Thanks Rebeca! I'm going to fix it :D